framework/bootstrap/configcontroller.go

```diff
@@ -35,70 +35,64 @@ type ConfigController interface {
 	viewstore.ViewerStore
 	RegisterEventHandler(kind resource.GroupVersionKind, handler func(resource.Config, resource.Config, Event))
 	Run(stop <-chan struct{})
+	// Ready return all controller has ready
```

Comment:
It is confusing to have `all controller` in the `ConfigController` interface method comment, and it is better for me to only reflect the state of this `ConfigController` itself.